requirements.txt

```diff
@@ -1,35 +1,37 @@
-pip==8.1.1
-django==3.2.8
-Markdown~=2.6.6
-celery==5.2.3
-celery-haystack
-django-haystack
-django-celery-email
-django-taggit==0.18.1
-pytz==2021.3
-requests==2.11.1
-django-floppyforms==1.6.2
-python-magic==0.4.6
-django-tastypie==0.13.3
-python-mimeparse>=0.1.4
-django-configurations==2.3.2
-django-storages==1.12.3
-dj-database-url==0.5.0
-django-cache-url==3.3.0
-django-compressor==3.1
+django==4.2.4
+Markdown==3.4.3
+celery==5.2.7
+django-celery-email==3.0.0
+django-taggit==4.0.0
+pytz==2023.3
+requests==2.31.0
+django-floppyforms==1.9.0
+python-magic==0.4.27
+python-mimeparse==1.6.0
+django-configurations==2.4.1
+django-storages==1.13.2
+dj-database-url==2.0.0
+django-compressor==2.1
 django-contrib-comments==2.2.0
 unicodecsv==0.14.1
-honcho==0.6.6
-django-tinymce==3.4.0
-anyjson==0.3.3
-pysolr==3.5.0
-python-docx==0.8.5
-elasticsearch==2.3.0
+django-tinymce==3.6.1
+python-docx==0.8.11
+elasticsearch==8.11.1
 
--e git+https://github.com/codeforhawaii/froide.git@v2#egg=froide
+-e git://github.com/codewithaloha/froide.git@main#egg=froide
```

Comment:
The protocol portion of the URL should be *git+https* otherwise the `pip install` command will timeout and fail on this dependency.

Reply:
Changed it and a quick `pip install -r requirements.txt` worked still 👍